### PR TITLE
companion for substrate#13883

### PR DESCRIPTION
**Pull request by andresilva**
cumulus companion: https://github.com/paritytech/cumulus/pull/2460

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2668738

File: node/service/src/lib.rs [hunk 1]
```diff
@@ -552,9 +552,6 @@ where
 		Some(shared_authority_set.clone()),
 	);
 
-	let shared_epoch_changes = babe_link.epoch_changes().clone();
-	let slot_duration = babe_config.slot_duration();
```
Comment (bkchr) [inline node/service/src/lib.rs:556]: We still need this?

**Review by bkchr — APPROVED** [review]

File: node/service/src/lib.rs (hunk 1 above)
Comment (bkchr) [reply node/service/src/lib.rs:556]: I mean, you removed `slot_duration` and the code still needs it, so it will fail to compile or?

Comment (acatangiu) [reply node/service/src/lib.rs:556]: `slot_duration` initialized at [L524](https://github.com/paritytech/polkadot/blob/20906a4f7778a3645815c891a39b92043d113795/node/service/src/lib.rs#L524)

**Review by acatangiu — APPROVED** [review]

File: node/service/src/lib.rs (hunk 1 above)
Comment (andresilva) [reply node/service/src/lib.rs:556]: I thought you meant in general if we still needed this in `babe_config` 😅 Yes, `slot_duration` was duplicated in this function.

Comment (bkchr) [reply node/service/src/lib.rs:556]: Ahh, I should have checked all the code! ty @acatangiu

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot/-/jobs/2694428

**Pull request merged**